ts/packages/ts/src/commands/UpsertCommand.ts

```diff
@@ -66,6 +101,8 @@ export class UpsertCommand extends Command<void> {
       })
       .value();
 
-    await store.set(upsertConfigs);
+    const configsToUpsert = _.unionBy(upsertConfigs, pipeConfigs, 'key');
```

Comment:
not a good naming here! this is a the same name as in the :92.
MergedConfigsToUpsert